pkg/koordlet/util/system/kidled_util.go

```diff
@@ -29,8 +29,14 @@ import (
 )
 
 var (
-	isSupportColdMemory *atomic.Bool = atomic.NewBool(false)
-	isStartColdMemory   *atomic.Bool = atomic.NewBool(false)
+	isSupportColdMemory              *atomic.Bool = atomic.NewBool(false)
+	isStartColdMemory                *atomic.Bool = atomic.NewBool(false)
+	kidledColdBoundary                            = defaultKidledColdBoundary
+	kidledHotBoundary                             = defaultKidledHotBoundary
+	defaultKidledScanPeriodInseconds uint32       = 5
+	defaultKidledUseHierarchy        uint8        = 1
+	defaultKidledColdBoundary        int          = 3
+	defaultKidledHotBoundary         int          = 3
```

Comment:
Perhaps the non-cold pages are considered "hot". Please describe the use cases to avoid over-design if an additional threshold is added.